content/docs/tools/pack/cli/install.md

```diff
@@ -71,15 +93,6 @@ docker run \
 
 ---
 
-### Windows
-To install `pack` on Windows, the easiest way is to use Chocolatey:
-```
-choco install pack --version={{< latest >}}
-```
-
-Alternatively, you can install the Windows executable for `pack` by downloading the Windows [ZIP file](https://github.com/buildpacks/pack/releases/download/v{{< latest >}}/pack-v{{< latest >}}-windows.zip).
-
-<hr/>
 
 # Auto-completion
```

Comment:
Now that pack completion works on both zsh and bash, perhaps we should just update this to reflect that?